contentgrid-appserver-autoconfigure/src/main/java/com/contentgrid/appserver/autoconfigure/actuator/ContentgridActuatorAutoConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.contentgrid.appserver.autoconfigure.actuator;
+
+import com.contentgrid.appserver.actuator.ActuatorConfiguration;
+import com.contentgrid.appserver.actuator.policy.PolicyActuator;
+import com.contentgrid.appserver.autoconfigure.security.DefaultSecurityConfiguration;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication.Type;
+import org.springframework.context.annotation.Import;
+
+@AutoConfiguration
+@ConditionalOnClass({PolicyActuator.class})
```

Comment:
It might be more logical to make it conditional like this, because you're importing it later
```suggestion
@ConditionalOnClass({ActuatorConfiguration.class})
```